src/app/pages/subjects/subjects.js

```diff
@@ -0,0 +1,149 @@
+import BaseView from '~/helpers/backbone/view';
+import BaseModel from '~/helpers/backbone/model';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './subjects.hbs';
+import FilterButton from './filter-button/filter-button';
+import CategorySection from './category-section/category-section';
+
+function makeBook(title, categories) {
+    return {
+        title,
+        categories,
+        coverUrl: `https://placeholdit.imgix.net/~text?txtsize=33&txt=${title}&w=200&h=180`
+    };
+}
+
+@props({
+    template: template,
+    regions: {
+        filterButtons: '.filter-buttons',
+        bookViewer: '.book-viewer'
+    }
+})
+export default class Subjects extends BaseView {
+
+    constructor() {
+        super();
+        this.model = new BaseModel({viewIsReady: false});
```

Comment:
The Model should have no knowledge of the state of the view.  The view's state should be stored by the view itself.  Otherwise it's breaking the separation of concerns.